app/src/environments/environment.ts

```diff
@@ -24,6 +24,12 @@ export const environment = {
             // https://github.com/center-for-threat-informed-defense/attack-workbench-collection-manager
             enabled: true, //if false, all systems for collection management will be disabled
             url: "http://localhost:3001/cm-api"
+        },
+        attack_website: {
+          // configuration for the ATT&CK Website
+          // https://github.com/mitre-attack/attack-website
+          enabled: true, // if false, the attack website integration will not work
```

Comment:
`enabled` should default to `false` since this is an optional configuration that can be set if there is a local instance of the website running.